css/styles.css

```diff
@@ -12,5 +12,9 @@ body {
   font-family: Helvetica, sans-serif;
   font-size: 1.6rem;
   color: #222;
-  background: #efefef;
+  background: khaki;
```

Comment:
@Kardinalkay, great addition as colours match the theme of website. Nevertheless, the background rule is repeated. Will be commented out or likely reapplied to some other elements when the code base grows in future version because it's a neutral colour.